src/main/scala/com/microsoft/hyperspace/index/IndexConstants.scala

```diff
@@ -47,4 +47,8 @@ object IndexConstants {
     val PLAIN_TEXT = "plaintext"
     val HTML = "html"
   }
+
+  val DATA_FILE_NAME_COLUMN = "_data_file_name"
+  val INDEX_LINEAGE_ENABLED = "spark.hyperspace.index.lineage.enabled"
+  val INDEX_LINEAGE_ENABLED_DEFAULT = "false"
```

Comment:
For now I have disabled lineage for indexes - I already made sure that all test cases pass irrespective of this default value (i.e. all tests pass with this default value be true or false).